### ISSUE TITLE
AV when using CopyBitmap32ToClipboard() with large images

### ISSUE THREAD

**Issue by huyi7** [graphics32/graphics32#258]
Hi, I was testing clipboard with large images like 5k x 5k+ using CopyBitmap32ToClipboard() in GR32_Clipboard.pas. I sometimes get AV from the line 202 of GR32_Clipboard.pas:
      System.Move(Buffer, Pointer(Longint(FPointer) + Position)^, Count); 

If I change the typecast to NativeUInt:
      System.Move(Buffer, Pointer(NativeUInt(FPointer) + Position)^, Count);  

it seems to work every time.  Looks like the AV was due to the typecast.

**Comment by andersmelander** [comment]
Yes, `LongInt` is incorrect. It will work on 32-bit but fail on 64-bit.

**Comment by huyi7** [comment]
Thanks a lot for your quick action!
